components/admin-tabs/Applicants.tsx

```diff
@@ -149,20 +150,30 @@ const Applicants: React.FC = () => {
     };
   });
 
+  const ExportButton: React.FC<{ cb: typeof downloadApplicationCsv; text: string }> = ({
+    cb,
+    text,
+  }) => {
+    return (
+      <Tooltip overlay={'Data exports respect currently applied filter and sort.'}>
+        <Button
+          className="export-button"
+          type="primary"
+          loading={!data || exporting}
+          onClick={onExportClick(cb)}
```

Comment:
basically this just makes the download function a parameter, `cb` is short for `callback`